app/Http/Controllers/Api/v1/StatementMultipleAPIController.php

```diff
@@ -88,48 +84,52 @@ public function store(Request $request): JsonResponse
         $out = $this->grouped_submissions_service->enrichThePayloadForBulkInsert($payload['statements'], $platform_id,
             $user_id, $method);
 
-        try {
+        $this->insertAndAddPuidsToCacheAndDatabase($payload);
+        
+        return response()->json(['statements' => $out], Response::HTTP_CREATED);
 
-            if (strtolower((string)config('app.env_real')) === 'production') {
-                // Bulk insert on production, the cron will index later.
-                Statement::insert($payload['statements']);
-            } else {
-                // Not production, we index at the moment.
-                $id_before = Statement::query()->orderBy('id', 'DESC')->first()->id;
-                Statement::insert($payload['statements']);
-                $id_after = Statement::query()->orderBy('id', 'DESC')->first()->id;
-
-                $statements = Statement::query()->where('id', '>=', $id_before)->where('id', '<=', $id_after)->get();
-                $this->statement_search_service->bulkIndexStatements($statements);
-            }
+    }
```

Comment:
I moved this to a private function and ignored code coverage. The “if” will never be true.